src/sources/soundsourceproxy.h

```diff
@@ -105,7 +105,7 @@ class SoundSourceProxy {
 
     /// Controls which (metadata/coverart) and how tags are (re-)imported from
     /// audio files when creating a SoundSourceProxy.
-    enum class ImportTrackMetadataMode {
+    enum class UpdateTrackFromSourceMode {
```

Comment:
```suggestion
    enum class UpdateTrackMetadataFromSourceMode {
```
?

Reply:
We have a class `TrackMetadata` that doesn't include cover art. Since we update the whole track including the cover info just `Track` is a better fit and it also closely mirrors the function name.

Reply:
Ok

Reply:
The distinction between the different levels of track properties and their naming (`TrackMetadata` < `TrackRecord`) could still be improved. But this is not urgent as long as the names are used consistently.